eva_submission/nextflow/accession.nf

```diff
@@ -133,7 +133,7 @@ process create_properties {
  * Accession VCFs
  */
 process accession_vcf {
-    clusterOptions "-g /accession/instance-${params.instance_id} \
+    clusterOptions "-g /accession \
```

Comment:
Why our we changing the accessioning instance? This is needed to prevent two accessioning job running at the same time.

Reply:
Sorry don't know what I was thinking... maybe got confused when I was looking at the other processes that use the accession group (to manage job tracker connections) but not the instance (to manage accessions)...